### PR TITLE
Upgrade packages

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

This upgrades NextJs to v12, Sanity to all of its latest versions and a whole bunch of minor upgrades to other libs.

### CODE REVIEW

File: packages/app/package.json [hunk 1]
```diff
@@ -104,18 +104,18 @@
     "cross-env": "^7.0.3",
     "decompress": "^4.2.1",
     "download": "^8.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-next": "^11.1.2",
+    "eslint": "^8.2.0",
```
Comment (VWSCoronaDashboard17) [inline packages/app/package.json:107]: AFAIK Next.js isn't compatible with this version yet and will show warnings during the build.

**Review by VWSCoronaDashboard17 — APPROVED** [review]

File: packages/app/package.json (hunk 1 above)
Comment (VWSCoronaDashboard1) [reply packages/app/package.json:107]: I'll check, I didn't notice them when testing but its very much possible I missed the warning

Comment (VWSCoronaDashboard1) [reply packages/app/package.json]: Ok, downgraded eslint to previous version

**Review by VWSCoronaDashboard14 — APPROVED** [review]

**Review by VWSCoronaDashboard17 — APPROVED** [review]

**Comment by VWSCoronaDashboard17** [comment]
🚂

**Pull request merged**